entity-framework/core/get-started/xamarin.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Getting Started - EF Core
+author: masoucou
+ms.date: 07/07/2020
+uid: core/get-started/xamarin
+---
+
+# Getting Started with EF Core and Xamarin
+
+In this tutorial, you create a Xamarin.Forms application that performs data access against a SQLite database using Entity Framework Core.
+
+You can follow the tutorial by using Visual Studio on Windows or Visual Studio for Mac.
+
+[View this article's sample on GitHub](https://github.com/dotnet/EntityFramework.Docs/tree/master/samples/core/GetStarted).
```

Comment:
We generally put this in Tip Like this https://docs.microsoft.com/en-us/ef/core/querying/